{"hierarchy":{"paths":[["doc:\/\/com.auth0.Auth0\/documentation\/Auth0","doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError"]]},"variants":[{"paths":["\/documentation\/auth0\/authenticationerror\/isrefreshtokendeleted"],"traits":[{"interfaceLanguage":"swift"}]}],"abstract":[{"type":"text","text":"When the credentials renewal fails because the user was deleted."}],"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"isRefreshTokenDeleted","kind":"identifier"},{"text":": ","kind":"text"},{"text":"Bool","preciseIdentifier":"s:Sb","kind":"typeIdentifier"},{"kind":"text","text":" { "},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}],"platforms":["iOS"],"languages":["swift"]}],"kind":"declarations"}],"sections":[],"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/isRefreshTokenDeleted"},"schemaVersion":{"major":0,"minor":3,"patch":0},"metadata":{"externalID":"s:5Auth019AuthenticationErrorV21isRefreshTokenDeletedSbvp","symbolKind":"property","roleHeading":"Instance Property","role":"symbol","title":"isRefreshTokenDeleted","modules":[{"name":"Auth0"}],"fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"isRefreshTokenDeleted","kind":"identifier"},{"kind":"text","text":": "},{"text":"Bool","kind":"typeIdentifier","preciseIdentifier":"s:Sb"}]},"references":{"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError/isRefreshTokenDeleted":{"type":"topic","url":"\/documentation\/auth0\/authenticationerror\/isrefreshtokendeleted","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError\/isRefreshTokenDeleted","title":"isRefreshTokenDeleted","abstract":[{"type":"text","text":"When the credentials renewal fails because the user was deleted."}],"role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"isRefreshTokenDeleted","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Bool","preciseIdentifier":"s:Sb"}],"kind":"symbol"},"doc://com.auth0.Auth0/documentation/Auth0":{"url":"\/documentation\/auth0","type":"topic","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0","title":"Auth0","abstract":[{"type":"text","text":"SDK for Apple platforms."}],"kind":"symbol","role":"collection"},"doc://com.auth0.Auth0/documentation/Auth0/AuthenticationError":{"navigatorTitle":[{"text":"AuthenticationError","kind":"identifier"}],"role":"symbol","identifier":"doc:\/\/com.auth0.Auth0\/documentation\/Auth0\/AuthenticationError","title":"AuthenticationError","type":"topic","kind":"symbol","fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"kind":"identifier","text":"AuthenticationError"}],"abstract":[{"type":"text","text":"Represents an error during a request to the Auth0 Authentication API."}],"url":"\/documentation\/auth0\/authenticationerror"}}}